queries.yaml

```diff
@@ -1052,7 +1052,7 @@ post_process:
       end_zoom: 9
       items_matching: { kind: locality }
       max_items: 1
-      grid_width: 3
+      grid_width: 24
```

Comment:
Try `12` (which is 4 times original value of 3)

Reply:
Fixed in 50bb8982.